src/content/docs/apm/new-relic-apm/maintenance/remove-applications-new-relic.mdx

```diff
@@ -78,6 +78,15 @@ Before you can remove an application monitored by New Relic APM, browser monitor
     * <DNT>**Remove it using the UI.**</DNT> Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Application**</DNT>, and click <DNT>**Delete application**</DNT>.
     * <DNT>**Use NerdGraph to delete an entity.**</DNT> If you've done the above and are still seeing that app in the UI, you can [use NerdGraph to delete the relevant entities](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial/#delete-entities). For how to find entity IDs, see [Entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/#find).
   </Collapser>
+
+  <Collapser
+    id="remove-otel-apps"
+    title="OpenTelemetry"
+  >
+    After 93 days, the app will be removed from the UI. If you want to remove it sooner than that, you will need to use the NerdGraph API:
+
+    <DNT>**Use NerdGraph to delete an entity.**</DNT> If you've done the above and are still seeing that app in the UI, you can [use NerdGraph to delete the relevant entities](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial/#delete-entities). For how to find entity IDs, see [Entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/#find).
```

Comment:
Can we rewrite the new content as following:

After 93 days, the app will be removed from the UI. If you want to remove it sooner than that,  use [NerdGraph to delete the relevant entities](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial/#delete-entities). To find entity IDs, see [Entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/#find).